scrappers/newsScrapper.ts

```diff
@@ -0,0 +1,43 @@
+import * as cheerio from "cheerio";
+import { Element as CheerioElement, Cheerio, SelectorType } from "cheerio";
+
+export interface News {
+  id: number
+  title: string,
+  description: string,
+  content?: string,
+  photoUrl: string,
+  date: Date,
+}
+
+export async function scrapNews(): Promise<News[]> {
+  const data = await fetch("https://www.mat.umk.pl/wiadomosci/", { method: "GET" });
+  const $ = cheerio.load(await data.text());
+
+  const news: News[] = [];
+
+  $("section.wiadomosc").each((_, element) => {
+    news.push(createNewsFromSectionTag($(element)));
+  });
+
+  return news;
+}
+
+export async function fillNewsContent(news: News): Promise<News> {
+    const data = await fetch(`https://www.mat.umk.pl/wiadomosci/?id=${news.id}`)
+    const $ = cheerio.load(await data.text());
+
+    news.content = $("article").html() ?? "";
+
+    return news;
+}
+
+function createNewsFromSectionTag(newsElement: Cheerio<CheerioElement>): News {
+  return {
+    id: parseInt(newsElement.find("p.wiecej a").attr("href")?.slice(4) ?? ""),
+    title: newsElement.find("h3").text(),
```

Comment:
Przydałaby się jakaś obsługa błędów, co się stanie gdy np. nie znajdziemy h3? Dobrze żeby kod był odporny na mikro zmiany po stronie frontu

Reply:
Jedną z rzeczy jaką zrobiłem to po prostu wzięcie defaultowych wartości. Jeszcze przyjrzę się temu.

Reply:
Pozmieniałem trochę selektory, jak tagu danego nie będzie to po prostu będzie pusty string. 
Wątpie aby zmieniali strukturę tej strony w ciągu najbliższych 100 lat.